bin/lb-ng.js

```diff
@@ -40,7 +40,9 @@ assertLoopBackVersion();
 if (app.booting) {
```

Comment:
I think I have encountered this issue while working on loopback-next too. Apparently, `loopback-boot@3` is setting the `app.booting` flag a later turn of the event loop.

The flag can have one of the following values:
- `undefined`: boot has not started yet or is in (early) progress
- `true`: boot is in progress
- `false`: boot was finished

Instead of adding a hard-coded timeout that will just slow down the process for too long in most cases, I am proposing to fix the logic detecting whether an app is booting.

```suggestion
 if (app.booting !== false) {
```

Reply:
I'm checking this solution

Reply:
@bajtos it works great! Many thanks 😄